cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh

```diff
@@ -66,54 +66,36 @@ export CONSUL_LOCAL_CONFIG
 
 echo "$(date) CONSUL_LOCAL_CONFIG: ${CONSUL_LOCAL_CONFIG}"
 
-echo "$(date) ENABLE_REGISTRY_ACL = ${ENABLE_REGISTRY_ACL}"
+echo "$(date) Starting edgex-core-consul with ACL enabled ..."
```

Comment:
we should remove edgex- prefix

Reply:
Hi @cloudxxx8 ,  right now the Consul's ACL policy we use is using `edgex` as prefix because the node and agent name of Core consul has the prefix `edgex` so we stick to it for now. See details:   https://github.com/edgexfoundry/edgex-go/blob/master/internal/security/bootstrapper/command/setupacl/aclpolicies.go#L39-L47

Reply:
@cloudxxx8 , @jim-wang-intel , also this is not a `service key` , rather the service name...

Reply:
no problem

Reply:
ok, although i don't the difference between service key and service name, to be honest ;-)